$MirOS: ports/www/lynx/patches/patch-lynx_help_Lynx_users_guide_html,v 1.2 2007/05/20 12:06:19 tg Exp $

	site policy

--- lynx_help/Lynx_users_guide.html.orig	Thu Aug  2 23:24:04 2007
+++ lynx_help/Lynx_users_guide.html	Fri Aug  3 12:00:19 2007
@@ -572,7 +572,7 @@ and is accessed by setting FORMS_OPTIONS
 
      (E)ditor                     : emacs
      (D)ISPLAY variable           : aixtest.cc.ukans.edu:0.0
-     mu(L)ti-bookmarks: OFF       B)ookmark file: lynx_bookmarks.html
+     mu(L)ti-bookmarks: OFF       B)ookmark file: ./.etc/bookmark.htm
      (F)TP sort criteria          : By Filename
      (P)ersonal mail address      : montulli@netscape.com
      (S)earching type             : CASE INSENSITIVE
@@ -2487,7 +2487,7 @@ where
                 <dd>Sets the connection timeout, where N is given in seconds.
              <dt><code>-cookie_file=FILENAME</code>
                 <dd>specifies a file to use to read cookies.
-                    If none is specified, the default value is ~/.lynx_cookies
+                    If none is specified, the default value is ~/.etc/cookies.txt
                     for most systems, but ~/cookies for MS-DOS.
              <dt><code>-cookie_save_file=FILENAME</code>
                 <dd>specifies a file to use to store cookies.
